docs/source/sci_api_tutorial/mutate_stru.rst

```diff
@@ -0,0 +1,189 @@
+==============================================
+Mutate Structure
+==============================================
+
+Briefs
+==============================================
+The target for this tutorial to mutate the target protein at defined residue(s). The main function provided are ``assign_mutant()`` and ``mutate_stru()``, in which ``mutate_stru()`` solves the problem of protein structural prediction upon mutation. It means to determine which mutation to address and determine the structure of the mutant of the source protein caused by residue substitution, deletion, and insertion. (see also: structure_prediction module for an alternative solution)
+
+Input/Output
+==============================================
+
+.. panels::
+
+    :column: col-lg-12 col-md-12 col-sm-12 col-xs-12 p-2 text-left
+
+    .. image:: ../../figures/mutation_mutate_stru_io.svg
+        :width: 100%
+        :alt: assign_mutant_io 
+
+Input
+------------------------------------------------
+
+``stru``
+    the target protein structure intended for mutation, represented as a  ``Structure()`` object.
+
+    .. admonition:: How to obtain
+
+        | A Strutcure() object can be obtained by these `APIs <obtaining_stru.html>`_.
+
+``mutation_list``
+    a target list of mutation objects, generated by the ``assign_mutant`` function.
+
+    .. admonition:: How to obtain
+
+        | A ``mutation_list()`` object can be obtained by these `APIs <assign_mutant.html>`_.
```

Comment:
Good effort!
However in this case, it is not "these APIs", but instead "assign_mutant()"